config/config.php

```diff
@@ -76,7 +76,7 @@
     | Supported drivers: "cache", "database"
     |
     */
-    'token_storage' => 'database',
+    'token_storage' => 'cache',
```

Comment:
Why did you change it, what is your reason? And you should update the readme doc, right?

Reply:
I think it's better to set the default driver to `cache` because it's easier and we'd rather use it over `database`. And yes, we need to update README as well. If we're on the same page with this, I'm gonna do the change.

Reply:
Ok, so after updating readme file please merge this PR. Thanks again 🙏🏻